gulpfile.js

```diff
@@ -103,8 +103,11 @@ gulp.task('copy:main.css', function () {
 
     return gulp.src(dirs.src + '/css/main.css')
                .pipe(plugins.header(banner))
-               .pipe(gulp.dest(dirs.dist + '/css'));
-
+                .pipe(plugins.autoprefixer({
+                    browsers: ['last 2 versions', 'ie >= 8'],
```

Comment:
> Btw, that’s the main reason AutoPrefixer still supports Opera 12 by default.

From https://github.com/ai/browserslist#readme, the [defaults](https://github.com/ai/browserslist/blob/fbe57b111e5b236e2be314d0840fa46deaa2e953/index.js#L86-L92) are:

`> 1%, last 2 versions, Firefox ESR, Opera 12.1.`